mono/mini/aot-compiler.c

```diff
@@ -7375,6 +7376,39 @@ add_gsharedvt_wrappers (MonoAotCompile *acfg, MonoMethodSignature *sig, gboolean
 	}
 }
 
+static void mono_destroy_compile_aot (MonoAotCompile *acfg)
+{
```

Comment:
This should have a newline after void.
There is no need to prefix static functions with 'mono_'.